src/style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.search {
+  position: relative;
+
+}
+
+.search-field {
+  padding-left: 62px;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 70px;
+  height: 70px;
+  display: block;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search::before {
```

Comment:
<img width="1100" alt="Screenshot 2022-04-01 at 11 14 36" src="https://user-images.githubusercontent.com/37452763/161223507-be04db35-90e6-428a-8ea0-b79186f59595.png">

try to make the icon bigger and add `background-repeat: no-repeat;`
also, center your icon using the `top` and `left` rules.

Reply:
Thanks for the advice, I added ` background-repeat: no-repeat;`
The icon for some reason is displayed completely only in the size  like for big SearchBar (19x19px). So I made the same icon size also for small SearchBar.
I left top and left rules like in the mockup.

Reply:
I fixed this problem with `background-repeat: no-repeat;`, thanks for the advice!